### PR TITLE
Print user-friendly error if internet is down

### PR DESCRIPTION
**Pull request by SamVerschueren**
Yesterday my internet seemed very slow so I used `speed-test` to test it. It seemed my internet connection was down entirely and it couldn't reach the speed-test servers.

```
{ [Error: getaddrinfo ENOTFOUND www.speedtest.net www.speedtest.net:80]
  code: 'ENOTFOUND',
  errno: 'ENOTFOUND',
  syscall: 'getaddrinfo',
  hostname: 'www.speedtest.net',
  host: 'www.speedtest.net',
  port: 80 }
```

So I thought it would be a good idea to show a more user-friendly error if this happens. Feel free to suggest another error message or to not merge at all :).

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -133,6 +133,10 @@ st.on('done', function () {
 });
 
 st.on('error', function (err) {
-	console.error(err);
+	if (err.code === 'ENOTFOUND') {
+		console.log(chalk.bold.red('Error:'), 'please check your internet connection.');
```
Comment (sindresorhus) [inline cli.js]: Should also be `console.error`

Comment (sindresorhus) [inline cli.js]: > Please check your internet connection

File: cli.js [hunk 2]
```diff
@@ -133,6 +133,10 @@ st.on('done', function () {
...
+	} else {
+		console.error(err);
+	}
```
Comment (sindresorhus) [inline cli.js]: Empty line here

File: cli.js (hunk 1 above)
Comment (sindresorhus) [inline cli.js]: Instead of `Error:`, use https://github.com/sindresorhus/log-symbols#error

**Comment by sindresorhus** [comment]
Good idea! I'm always super happy about making things more user-friendly. Please do open issues on my other projects too if you notice anything that could be more user-friendly ;)

File: cli.js (hunk 1 above)
Comment (SamVerschueren) [inline cli.js]: How much improvements can you have at so minimal code change :).

Thanks! All valid points off course.

File: cli.js [hunk 3]
```diff
@@ -133,6 +134,11 @@ st.on('done', function () {
 });
 
 st.on('error', function (err) {
-	console.error(err);
+	if (err.code === 'ENOTFOUND') {
+		console.error(logSymbols.error, 'Please check your internet connection');
```
Comment (SamVerschueren) [inline cli.js]: Capital `P` in the error message?

Comment (sindresorhus) [inline cli.js]: Yes

**Comment by sindresorhus** [comment]
Thanks Sam :)

**Comment by vasco3** [comment]
just wondering, why was this PR closed? kind of discouraging for future contributions if there is no feedback on why it was rejected

**Comment by sindresorhus** [comment]
@vasco3 Look at the commit log: https://github.com/sindresorhus/speed-test/commits/master I just manually squashed and merged it to keep a clean commit history.

**Comment by sindresorhus** [comment]
Also, see the above:

screen shot 2015-10-31 at 01 23 48

**Comment by SamVerschueren** [comment]
No problem! Always happy to contribute to your repos ;) :beers:

**Comment by vasco3** [comment]
oh ok thanks :)

**Pull request closed without merging**